src/SOS/lldbplugin/soscommand.cpp

```diff
@@ -186,11 +187,11 @@ sosCommandInitialize(lldb::SBDebugger debugger)
     g_services->AddCommand("dumpvc", new sosCommand("DumpVC"), "Displays info about the fields of a value class.");
     g_services->AddCommand("eeheap", new sosCommand("EEHeap"), "Displays info about process memory consumed by internal runtime data structures.");
     g_services->AddCommand("eestack", new sosCommand("EEStack"), "Runs dumpstack on all threads in the process.");
-    g_services->AddCommand("eeversion", new sosCommand("EEVersion"), "Displays information about the runtime version.");
+    g_services->AddCommand("eeversion", new sosCommand("EEVersion"), "Displays information about the runtime and SOS versions.");
     g_services->AddCommand("ehinfo", new sosCommand("EHInfo"), "Displays the exception handling blocks in a jitted method.");
     g_services->AddCommand("finalizequeue", new sosCommand("FinalizeQueue"), "Displays all objects registered for finalization.");
     g_services->AddCommand("findappdomain", new sosCommand("FindAppDomain"), "Attempts to resolve the AppDomain of a GC object.");
-    g_services->AddCommand("findroots", new sosCommand("FindRoots"), "");
+    g_services->AddCommand("findroots", new sosCommand("FindRoots"), "Find object roots across GC collections.");
```

Comment:
```suggestion
    g_services->AddCommand("findroots", new sosCommand("FindRoots"), "Finds and displays object roots across GC collections.");
```

Reply:
It seems it would make sense to have each of the verbs be of the same tense (with an s on the end).